cpp/net/url_fetcher.cc

```diff
@@ -162,7 +177,7 @@ void State::RequestDone(evhttp_request* req) {
     return;
```

Comment:
Does evhtp ever give us those silly null requests? I hope not, they're the first thing I'd have changed if I made a new library to replace evhttp... :stuck_out_tongue: